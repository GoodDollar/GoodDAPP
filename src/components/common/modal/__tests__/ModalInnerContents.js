import React from 'react'
import renderer from 'react-test-renderer'
import ImportedModalInnerContents from '../ModalInnerContents'
import { withThemeProvider } from '../../../../__tests__/__util__'
const ModalInnerContents = withThemeProvider(ImportedModalInnerContents)

// Note: test renderer must be required after react-native.

describe('ModalInnerContents', () => {
  it('matches snapshot', async () => {
    let component

    await renderer.act(
      // eslint-disable-next-line require-await
      async () =>
        (component = renderer.create(
          <ModalInnerContents>
            <React.Fragment>Testing</React.Fragment>
          </ModalInnerContents>,
        )),
    )
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })

  it('matches snapshot with properties', async () => {
    let component

    await renderer.act(
      // eslint-disable-next-line require-await
      async () =>
        (component = renderer.create(
          <ModalInnerContents>
            <React.Fragment>Testing</React.Fragment>
          </ModalInnerContents>,
        )),
    )
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
