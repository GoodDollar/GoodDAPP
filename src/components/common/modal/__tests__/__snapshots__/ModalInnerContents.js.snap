// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ModalInnerContents matches snapshot 1`] = `
<div
  className="css-view-1dbjc4n r-flex-13awgt0 r-pointerEvents-12vffkv"
>
  <div
    className="css-view-1dbjc4n r-backgroundColor-14lw9ot r-borderBottomRightRadius-qpmf2f r-borderTopRightRadius-dfv94e r-boxShadow-172poub r-flexGrow-16y2uox r-padding-nsbfu8 r-position-bnwqim r-zIndex-184en5c"
  >
    Testing
  </div>
</div>
`;

exports[`ModalInnerContents matches snapshot with properties 1`] = `
<div
  className="css-view-1dbjc4n r-flex-13awgt0 r-pointerEvents-12vffkv"
>
  <div
    className="css-view-1dbjc4n r-backgroundColor-14lw9ot r-borderBottomRightRadius-qpmf2f r-borderTopRightRadius-dfv94e r-boxShadow-172poub r-flexGrow-16y2uox r-padding-nsbfu8 r-position-bnwqim r-zIndex-184en5c"
  >
    Testing
  </div>
</div>
`;
